flink-connector-pulsar/archunit-violations/f4d91193-72ba-4ce4-ad83-98f780dce581

```diff
@@ -10,9 +10,3 @@ org.apache.flink.connector.pulsar.source.PulsarSourceITCase does not satisfy: on
 * reside in a package 'org.apache.flink.runtime.*' and is annotated with @ExtendWith with class InternalMiniClusterExtension\
 * reside outside of package 'org.apache.flink.runtime.*' and is annotated with @ExtendWith with class MiniClusterExtension\
  or contain any fields that are public, static, and of type MiniClusterWithClientResource and final and annotated with @ClassRule or contain any fields that is of type MiniClusterWithClientResource and public and final and not static and annotated with @Rule
-org.apache.flink.connector.pulsar.source.PulsarUnorderedSourceITCase does not satisfy: only one of the following predicates match:\
```

Comment:
That might be out of scope, but: There's another (empty) violations file. I haven't worked with archunit that much. I'm just wondering whether the empty file should be removed? :thinking:

Reply:
I don't know. I just leave these empty files. They are auto generated.